docs/source/overview.rst

```diff
@@ -13,25 +13,49 @@ Cap provides:
 1. energy storage device models
 2. electrochmical measurement techniques
 
-.. testcode::
+Guidelines for installation are :ref:`available <installation>`.
+Note that is **not** necessary to build Cap from source to use it.
+For instructions on how to use Cap without installing it, refer to the
+following :ref:`section <docker>`.
 
-    >>> from pycap import PropertyTree,EnergyStorageDevice
-    >>> input_database=PropertyTree()
-    >>> input_database.parse_xml('super_capacitor.xml')
-    >>> device=EnergyStorageDevice(input_database.get_child('device'))
 
+.. _docker:
 
 Alternative to the full install procedure
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 All you need is a working installation of Docker.
-
-The following command starts a Docker container with a Jupyter Notebook server listening for HTTP connections on port 8888.
+Follow the `Docker Engine installation guide
+<https://docs.docker.com/engine/installation/>`_ for details on how to
+install it on your machine.
+It is supported on Linux, Cloud, Windows, and OS X.
+
+The following command starts a Docker container with a Jupyter Notebook server
+listening for HTTP connections on port 8888.
+It mounts the present working directory, ``$PWD``, into the container at
+``/notebooks``, which is set as the Jupyter Notebook startup folder.
 It has pycap already installed on it and comes with a few notebooks as example.
+
 ::
 
-    docker run -d -p 8888:8888 dalg24/cap
+    $ docker run -d \
+          -p 8888:8888 \
+          -v $PWD:/notebooks \
+          dalg24/cap
+
```

Comment:
Maybe say that for ubuntu you need to use ```sudo```? It is the installation reference that you link but I am sure that people will just copy past the command and wonder why it doesn't work.